blocks/components/editable/tinymce.js

```diff
@@ -3,7 +3,13 @@ export default class TinyMCE extends wp.element.Component {
 		this.initialize();
 	}
 
-	shouldComponentUpdate() {
+	shouldComponentUpdate( nextProps ) {
+		const isEmpty = String( nextProps.isEmpty );
+
+		if ( this.editorNode.getAttribute( 'data-is-empty' ) !== isEmpty ) {
+			this.editorNode.setAttribute( 'data-is-empty', isEmpty );
+		}
```

Comment:
I'm not sure why this is needed, but is this function the right place for this? Should this be moved to another lifecycle hook? `componentDidUpdate` or `componentWillReceiveProps`?